rosdep/python.yaml

```
@@ -1180,6 +1180,16 @@ python-colorama:
     pip:
       packages: [colorama]
   ubuntu: [python-colorama]
+python-colorcet:
```

You probably want to make it available for python3, in which case it should be:

```suggestion
python3-colorcet:
```

---

sorry, I overlooked the description. 
I modified in [502c620](https://github.com/ros/rosdistro/pull/33681/commits/502c620d26d998c05d665ba2fa947fa3df345b52).